### PR TITLE
[rapids] Versions update: RAPIDS Spark 0.2.0, cuDF 0.15 and CUDA 11

### PR DESCRIPTION
**Pull request by sameerz**
Updating for RAPIDS Accelerator for Apache Spark 0.2, RAPIDS cudf 0.15 and CUDA 11.  

@mengdong for review

### CODE REVIEW

File: rapids/README.md [hunk 1]
```diff
@@ -138,15 +138,16 @@ To use XGBoost4j with Spark 2
 
 ### Step 1. Download dataset for Spark RAPIDS XGBoost application
 
-From [Spark examples](https://github.com/rapidsai/spark-examples) repository
+From [Spark examples](https://github.com/NVIDIA/spark-xgboost-examples/tree/spark-2) repository
```
Comment (medb) [inline rapids/README.md:141]: Should it be `.../spark-3`?

**Review by medb — CHANGES_REQUESTED** [review]

File: rapids/README.md (hunk 1 above)
Comment (sameerz) [reply rapids/README.md:141]: These examples are for XGBoost4j with Spark 2, pre-the Spark 3 release.  Once Spark 3 became the default branch, we needed to specify spark-2.

**Comment by sameerz** [comment]
@googlebot I signed it!

File: rapids/README.md (hunk 1 above)
Comment (sameerz) [reply rapids/README.md:141]: These examples are for XGBoost4j with Spark 2, pre-the Spark 3 release. Once Spark 3 became the default branch, we needed to specify the spark-2 branch for this example.

File: rapids/README.md [hunk 2]
```diff
@@ -129,24 +131,26 @@ To use XGBoost4j with Spark 2
     *   Multi-node clusters with homogenous GPU configuration
 *   Software Requirements
     *   NVIDIA driver 410.48+
-    *   CUDA v10.2/v10.1/v10.0/v9.2
-    *   NCCL 2.4.7 and later
-*   `EXCLUSIVE_PROCESS` must be set for all GPUs in each NodeManager (set by default in this
-    initialization action)
-*   `spark.dynamicAllocation.enabled` property must be set to `false` for Spark (set by default 
-    in this initialization action)
+    *   CUDA 9.2+
```
Comment (sameerz) [inline rapids/README.md:134]: This was tested with the previously listed versions, not yet tested with CUDA 11.  
```suggestion
    *   CUDA v10.2/v10.1/v10.0/v9.2
```

Comment (medb) [reply rapids/README.md]: Makes sense, reverted.

File: rapids/README.md [hunk 3]
```diff
@@ -20,19 +20,19 @@ initialization actions in production.
...
+*   Apache Spark 3.0+
 *   Hardware Requirements
     *   NVIDIA Pascal™ GPU architecture or better (V100, P100, T4 and later)
     *   Multi-node clusters with homogenous GPU configuration
 *   Software Requirements
-    *   NVIDIA driver 440.33+
-    *   CUDA v10.2/v10.1/v10.0
-    *   NCCL 2.4.7 and later
+    *   NVIDIA GPU driver 440.33+
+    *   CUDA 10.1+
```
[the remark this answers is not in the document]
Comment (medb) [reply rapids/README.md]: Makes sense, reverted.

**Review by medb — APPROVED** [review]

**Pull request merged**